### ISSUE TITLE
Reversed Sign?

### ISSUE THREAD

**Issue by whamza15** [pytorch/examples#298]
https://github.com/pytorch/examples/blob/963f7d1777cd20af3be30df40633356ba82a6b0c/vae/main.py#L105

Aren't we trying to maximize that and hence there needs to be a negative sign here?

**Comment by Kaixhin** [comment]
`loss_function` is formulated correctly for gradient descent - minimise the reconstruction error and minimise the KL divergence between the distributions.
